lms/templates/footer-edx-new.html

```diff
@@ -94,6 +94,22 @@ <h2 class="footer-follow-title">${_("Follow Us")}</h2>
           <span class="copy">${_("Google+")}</span>
         </a>
       </div>
+
+      % if settings.FEATURES.get('ENABLE_FOOTER_MOBILE_APP_LINKS'):
+      <div class="footer-mobile-apps">
+        <h2 class="footer-nav-title">${_("Mobile Apps")}</h2>
+        <div class="mobile-app-wrapper">
+          <a href="#">
```

Comment:
Will this link need to be filled in?

Also, maybe these links (including the one below for Google) should be retrieved from `settings` so ops can override them with a config change?